yellowbrick/classifier/classification_report.py

```diff
@@ -385,15 +396,24 @@ def classification_report(
         **kwargs
     )
 
-    # Create the train and test splits
-    X_train, X_test, y_train, y_test = train_test_split(
-        X, y, test_size=test_size, random_state=random_state
-    )
-
     # Fit and transform the visualizer (calls draw)
```

Comment:
@Kautumn06 taking a look at the test failures on Travis &mdash; looks like now that we've removed the splitting step from the quick method, we need to delete the unused import from the top of this file (`yellowbrick/classifier/classification_report.py`). Would you mind removing the line that reads `from sklearn.model_selection import train_test_split`?